frontend/src/__tests__/cypress/cypress/tests/mocked/modelServing/modelServingDeploy.cy.ts

```diff
@@ -287,6 +395,35 @@ describe('Model Serving Deploy Wizard', () => {
     cy.get('@createInferenceService.all').then((interceptions) => {
       expect(interceptions).to.have.length(2); // 1 dry-run request and 1 actual request
     });
+
+    //dry run request
+    cy.wait('@createRole').then((interception) => {
```

Comment:
I guess we only need to do the intercepting for one of the tests (just pick one of the tests to be an external and token testing one). The role looks good here but we also need to catch the rolebinding, secret and service account